lib/get-api.js

```diff
@@ -54,17 +54,18 @@ function getApi (state) {
     connectionStatus: hoodieConnectionStatus,
 
     // helpers
-    request: require('./request').bind(this, state),
+    request: require('./request').bind(null, state),
     log: log,
-    plugin: require('./plugin'),
 
     // events
-    on: require('./events').on.bind(this, state),
-    one: require('./events').one.bind(this, state),
-    off: require('./events').off.bind(this, state),
-    trigger: require('./events').trigger.bind(this, state)
+    on: require('./events').on.bind(null, state),
+    one: require('./events').one.bind(null, state),
+    off: require('./events').off.bind(null, state),
+    trigger: require('./events').trigger.bind(null, state)
```

Comment:
it makes no difference. But passing `null` makes clear that the `this` variable is not used inside the methods. We try to omit `this` altogether in all our code bases